### PR TITLE
fix: change checkout limit to 10.000

### PR DESCRIPTION
**Pull request by raulsteurer**
Valid values are now between 100 and 10.000 moneys

### CODE REVIEW

File: src/slices/ShopCheckout/messages.de.ts [hunk 1]
```diff
@@ -9,10 +9,8 @@ const messagesDe = {
   'sections.shopCheckout.contributionValue.unit.CHF': 'CHF',
   'sections.shopCheckout.contributionValue.validation.empty':
     'Bitte geben sie einen Betrag ein',
-  'sections.shopCheckout.contributionValue.validation.tooLow.EUR':
-    'Der Betrag muss mindestens 10 Euro sein',
-  'sections.shopCheckout.contributionValue.validation.tooLow.CHF':
-    'Der Betrag muss mindestens 10 CHF sein',
+  'sections.shopCheckout.contributionValue.validation.tooLow':
+    'Der Betrag muss mindestens 100 Kilogramm entsprechen',
```
Comment (rschiffer) [inline src/slices/ShopCheckout/messages.de.ts:13]:
```suggestion
    'Die Menge muss mindestens 100 Kilogramm entsprechen',
```

File: src/components/portfolio/SmallCheckout/messages.de.ts [hunk 2]
```diff
@@ -9,23 +9,16 @@ const messagesDe = {
...
   'portfolio.smallCheckout.contributionValueCurrency.unit.EUR': '€',
   'portfolio.smallCheckout.contributionValueCurrency.unit.CHF': 'Fr.',
-  'portfolio.smallCheckout.contributionValueCurrency.validation.tooLow.CHF':
-    'Der Wert muss mindestens 10 CHF betragen',
-  'portfolio.smallCheckout.contributionValueCurrency.validation.tooLow.EUR':
-    'Der Wert muss mindestens 10 Euro betragen.',
-  'portfolio.smallCheckout.contributionValueCurrency.validation.empty':
+
+  'portfolio.smallCheckout.contributionValueTons.validation.tooLow.CHF':
+    'Der Wert muss mindestens 100 Kilogramm betragen',
```
Comment (rschiffer) [inline src/components/portfolio/SmallCheckout/messages.de.ts:14]:
```suggestion
    'Die Menge muss mindestens 100 Kilogramm betragen',
```

File: src/constants/domain.ts [hunk 3]
```diff
@@ -1,3 +1,3 @@
-export const MINIMUM_CONTRIBUTION_VALUE_IN_MONEY = 10;
+export const MINIMUM_CONTRIBUTION_VALUE_IN_KG = 100;
 
-export const MAXIMUM_CONTRIBUTION_VALUE_IN_MONEY = 60_000;
+export const MAXIMUM_CONTRIBUTION_VALUE_IN_KG = 10_000;
```
Comment (lukasbals) [inline src/constants/domain.ts:3]: Are those the same min. and max. values as in the fpm-api?

Comment (raulsteurer) [reply src/constants/domain.ts:3]: Minimum yes, but the maximum is still set to a thousand tons.
I change that once the frontend has been updated.

File: src/components/portfolio/SmallCheckout/messages.de.ts [hunk 4]
```diff
@@ -16,16 +16,10 @@ const messagesDe = {
   'portfolio.smallCheckout.contributionValueCurrency.validation.empty':
     'Bitte geben Sie einen Wert ein',
   'portfolio.smallCheckout.contributionValueCurrency.validation.tooHigh':
-    'Für größere Einkäufe kontaktieren Sie bitte unser Verkaufsteam über den unten stehenden Button',
```
Comment (raulsteurer) [inline src/components/portfolio/SmallCheckout/messages.de.ts:19]: The button is optional and not necessarily there

**Review by lukasbals — APPROVED** [review]

**Review by rschiffer — APPROVED** [review]

**Pull request merged**